core/src/main/java/com/linecorp/armeria/client/cookie/DefaultCookieJar.java

```diff
@@ -122,28 +123,18 @@ public CookieState state(Cookie cookie, long currentTimeMillis) {
         requireNonNull(cookie, "cookie");
         lock.lock();
         try {
-            final Long createdTimeMillis = store.get(cookie);
-            if (createdTimeMillis == null) {
+            final long createdTimeMillis = store.getOrDefault(cookie, Long.MIN_VALUE);
+            if (createdTimeMillis == Long.MIN_VALUE) {
                 return CookieState.NON_EXISTENT;
             }
-            return isExpired(cookie, createdTimeMillis, currentTimeMillis) ?
+            final long timePassed = TimeUnit.MILLISECONDS.toSeconds(currentTimeMillis - createdTimeMillis);
```

Comment:
Should we increment by 1 if `(currentTimeMillis - createdTimeMillis) % 1000 != 0`? :thinking:

Reply:
Alternatively, we could convert maxAge to milliseconds instead of converting others to seconds.

Reply:
Right, to milliseconds is better 😄